neurokit2/eog/eog_plot.py

```diff
@@ -49,7 +49,10 @@ def eog_plot(eog_signals, peaks=None, sampling_rate=None):
 
     # Sanity-check input.
     if not isinstance(eog_signals, pd.DataFrame):
-        print("NeuroKit error: The `eog_signals` argument must be the DataFrame returned by `eog_process()`.")
+        raise ValueError(
```

Comment:
can add '*eog_plot()*' in "NeuroKit error: eog_plot(): The `eog_signals` ...."

Reply:
Good catch, will modify + add test